src/main/java/org/generationcp/ibpworkbench/ui/sidebar/WorkbenchSidebar.java

```diff
@@ -140,7 +137,7 @@ protected boolean doCollapse(TreeItem treeItem) {
 		return true;
 	}
 
-	protected ActionListener getLinkActions(final String toolName, Project project) {
+	public ActionListener getLinkActions(final String toolName, final Project project) {
```

Comment:
We can remove `MANAGE_PROGRAM `constant and its usage in the if clause now that `ToolName.isURLAccessibleTool` includes **manage_program** tool.